server/sonar-web/src/main/js/apps/overview/events/AnalysesList.js

```diff
@@ -24,12 +24,14 @@ import Analysis from './Analysis';
 import PreviewGraph from './PreviewGraph';
 import { getMetrics } from '../../../api/metrics';
 import { getProjectActivity } from '../../../api/projectActivity';
+import { getBranchName } from '../../../helpers/branches';
 import { translate } from '../../../helpers/l10n';
 /*:: import type { Analysis as AnalysisType } from '../../projectActivity/types'; */
 /*:: import type { History, Metric } from '../types'; */
 
 /*::
 type Props = {
+  branch: {},
```

Comment:
The branch parameters should also be passed down to `PreviewGraph` component, currently clicking on it while on a long lived branch display the activity graph for the main branch.